src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -0,0 +1,403 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+using DelimitedFiles: readdlm
+using LinearAlgebra: eigvals
+
+@muladd begin
+#! format: noindent
+
+# Abstract base type for both single/standalone and multi-level 
+# PERK (Paired-Explicit Runge-Kutta) time integration schemes
+abstract type AbstractPairedExplicitRK end
+# Abstract base type for single/standalone PERK time integration schemes
+abstract type AbstractPairedExplicitRKSingle <: AbstractPairedExplicitRK end
+
+# Compute the coefficients of the A matrix in the Butcher tableau using
+# stage scaling factors and monomial coefficients
+function compute_a_coeffs(num_stage_evals, stage_scaling_factors, monomial_coeffs)
+    a_coeffs = copy(monomial_coeffs)
+
+    for stage in 1:(num_stage_evals - 2)
+        a_coeffs[stage] /= stage_scaling_factors[stage]
+        for prev_stage in 1:(stage - 1)
+            a_coeffs[stage] /= a_coeffs[prev_stage]
+        end
+    end
+
+    return reverse(a_coeffs)
+end
+
+# Compute the Butcher tableau for a paired explicit Runge-Kutta method order 2
+# using a list of eigenvalues
+function compute_PairedExplicitRK2_butcher_tableau(num_stages, eig_vals, tspan,
+                                                   bS, c_end, verbose)
+
+    # c Vector form Butcher Tableau (defines timestep per stage)
+    c = zeros(num_stages)
+    for k in 2:num_stages
+        c[k] = c_end * (k - 1) / (num_stages - 1)
+    end
+    stage_scaling_factors = bS * reverse(c[2:(end - 1)])
+
+    # - 2 Since first entry of A is always zero (explicit method) and second is given by c_2 (consistency)
+    coeffs_max = num_stages - 2
+
+    a_matrix = zeros(coeffs_max, 2)
+    a_matrix[:, 1] = c[3:end]
+
+    consistency_order = 2
+
+    dtmax = tspan[2] - tspan[1]
+    dteps = 1e-9
+
+    num_eig_vals, eig_vals = filter_eig_vals(eig_vals, verbose)
```

Comment:
```suggestion
    num_eig_vals, eig_vals = filter_eig_vals(eig_vals; verbose)
```
If you make `verbose` a kwarg as suggested below